src/app/zap-templates/zcl/data-model/chip/access-control-cluster.xml

```diff
@@ -98,70 +107,101 @@ limitations under the License.
       <feature bit="0" code="EXTS" name="Extension" summary="Device provides ACL Extension attribute">
         <optionalConform/>
       </feature>
-      <feature bit="1" code="MNGD" name="Managed Device" summary="Device is managed">
-        <optionalConform/>
-      </feature>
+      <feature bit="1" code="MNGD" name="ManagedDevice" summary="Device is managed"/>
     </features>
     <server init="false" tick="false">true</server>
     <globalAttribute code="0xFFFD" side="either" value="2"/>
     <description>The Access Control Cluster exposes a data model view of a
       Node&apos;s Access Control List (ACL), which codifies the rules used to manage
       and enforce Access Control for the Node&apos;s endpoints and their associated
       cluster instances.</description>
-
     <attribute side="server" code="0x0000" define="ACL" type="array" entryType="AccessControlEntryStruct" writable="true">
       <description>ACL</description>
       <access op="read" privilege="administer"/>
       <access op="write" privilege="administer"/>
+      <mandatoryConform/>
     </attribute>
-
     <attribute side="server" code="0x0001" define="EXTENSION" type="array" entryType="AccessControlExtensionStruct" writable="true" optional="true">
       <description>Extension</description>
       <access op="read" privilege="administer"/>
       <access op="write" privilege="administer"/>
+      <mandatoryConform>
+        <feature name="EXTS"/>
+      </mandatoryConform>
+    </attribute>
+    <attribute side="server" code="0x0002" define="SUBJECTS_PER_ACCESS_CONTROL_ENTRY" type="int16u" min="4" default="4">
+      <description>SubjectsPerAccessControlEntry</description>
+      <mandatoryConform/>
+    </attribute>
+    <attribute side="server" code="0x0003" define="TARGETS_PER_ACCESS_CONTROL_ENTRY" type="int16u" min="3" default="3">
+      <description>TargetsPerAccessControlEntry</description>
+      <mandatoryConform/>
```

Comment:
Do we really want to be explicitly listing all mandatory things instead of just defaulting things mandatory unless stated otherwise?

Reply:
I think leaving mandatoryConform there will easier for paring XML data

Reply:
If you decide we should remove them, it should be an easy change in Alchemy

Reply:
I'm not sure what you mean by "parsing conformance from XML data".  I think it makes sense to say "conformance is M unless stated otherwise"...

In general, we need to decide whether the XML will always spell out all the things or just list them when they have non-default values. There are reasons to go in either direction, honestly, and tradeoffs in terms of what happens when a new "thing" is added vs how easy it is to extract the relevant information from the XML without having as separate list of defaults somewhere.

Either way, please consider how this is meant to actually get reviewed.  The bigger the deltas, the more time-intensive the review process.

Reply:
I agree that we should make the deltas smaller to ease the review process. Updating the entire XML introduces too large of a delta, so I'll switch to using Alchemy to only generate conformance data, which should make things more manageable.